### DISCUSSION TITLE
AI專利技術

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#616] [General]
我們在發展科技的同時，也需制定相關的規範，保護彼此也保護自己
https://udn.com/news/story/6871/6889517

**Comment by manipulateed** [comment]
本身就有資源有技術或是專業人員的公司或學校，將更能站在前面引領整個業界，但同樣的那些位於後面，能否抓得住前面拋出來的韁繩，抑或是他們都只自顧自地向前奔跑，造成斷層，可能就失去AI產業的多元競爭了。
